client/src/components/input/TextInput/TextInput.jsx

```diff
@@ -19,6 +19,12 @@ const TextInput = ({
   hasRestrictedInput,
   inputType,
 }) => {
+  useEffect(() => {
```

Comment:
Merge with dev (to avoid conflicts, if you haven't already), I had to add this change there for registration